docs/package.json

```diff
@@ -13,8 +13,8 @@
     "deploy-gh-pages": "gh-pages -d build"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-alpha.70",
-    "@docusaurus/preset-classic": "2.0.0-alpha.70",
+    "@docusaurus/core": "2.0.0-beta.8",
```

Comment:
Docusaurus v2 requires a particular version of Node (or higher), right? Can you add a note for that in the readme?